_posts/2020-09-16-uwbcode2020-announcement.md

```diff
@@ -12,6 +12,50 @@ exturl_uwbacm: https://uwbacm.com
 exturl_gwc: https://www.instagram.com/girlswhocode__uwb/?igshid=lmenj5wiuyo
 exturl_regisform: http://bit.ly/join-uwbcode20
 ---
+<!-- Display the countdown timer in an element -->
+<script>
+// Set the date we're counting down to
+var countDownDate = new Date("Oct 9, 2020 10:00:00 PDT").getTime();
+
+// Update the count down every 1 second
+var x = setInterval(function() {
```

Comment:
can you change this single letter variable name `x` to something more meaningful?